index.html

```diff
@@ -302,10 +302,10 @@ <h4 class="desktop-only">FACEBOOK</h4>
 
             <img class="escritorio-trabajo-foto-izquierda" src="media/escritoriotrabajo2.png" alt="">
          
-         </div>
+         </main>
 
 
-         <div class="trabajo tamaño flex-column ">
+         <main class="trabajo tamaño flex-column ">
```

Comment:
- I suggest you change the <main> tag to <section> as it is recommended by good practice for semantic structure, accessibility, styling, organization and maintainability of the code.